elliptic-curve/src/hash2curve/group_digest.rs

```diff
@@ -34,9 +37,9 @@ pub trait GroupDigest {
     /// let pt = ProjectivePoint::hash_from_bytes::<hash2field::ExpandMsgXof<sha3::Shake256>>(b"test data", b"CURVE_XOF:SHAKE-256_SSWU_RO_");
     /// ```
     ///
-    fn hash_from_bytes<X: ExpandMsg>(msg: &[u8], dst: &'static [u8]) -> Self::Output {
+    fn hash_from_bytes<X: ExpandMsg>(msg: &[u8], dst: &'static [u8]) -> Result<Self::Output> {
         let mut u = [Self::FieldElement::default(), Self::FieldElement::default()];
-        hash_to_field::<X, _>(msg, dst, &mut u);
+        hash_to_field::<X, _>(msg, dst, &mut u)?;
```

Comment:
Theoretically this can't fail if users are only using `ExpandMsg` specified in the spec and don't implement `FromOkm` on a curve with an incredibly high number, again, not specified in the spec.

WDYT? Should we unwrap? Honestly I'm fine either way.

Reply:
How about I merge this PR and you can open a follow-up to play with that?